compose.yaml

```diff
@@ -61,8 +61,9 @@ services:
       - KC_HOSTNAME_STRICT_HTTPS=false
       - KC_HTTP_RELATIVE_PATH=/auth
       - KC_PROXY=edge
+      - KC_PROXY_HEADERS=xforwarded
     # import: docker compose exec keycloak /opt/keycloak/bin/kc.sh import --override true --file /opt/keycloak/data/import/test-config.json
-    # export: docker compose run --rm keycloak /opt/keycloak/bin/kc.sh export --file /opt/keycloak/data/import/test-config.json
```

Comment:
Would make sense to put these 2 into Makefile.
Also the import command probably also doesnt need the kc.sh anymore?

Reply:
You are right. Done :+1: